openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -89,26 +122,25 @@ class AccessTokenView(_DispatchingView):
 
     def dispatch(self, request, *args, **kwargs):
         response = super().dispatch(request, *args, **kwargs)
-
-        token_type = request.POST.get('token_type',
-                                      request.META.get('HTTP_X_TOKEN_TYPE', 'no_token_type_supplied')).lower()
-        monitoring_utils.set_custom_attribute('oauth_token_type', token_type)
-        monitoring_utils.set_custom_attribute('oauth_grant_type', request.POST.get('grant_type', ''))
+        monitoring_utils.set_custom_attribute('oauth_grant_type', request.POST.get('grant_type', 'not-supplied'))
```

Comment:
**Note to Reviewer:** I think the only change during this refactor was updating the default value here.